ruby-challenges.rb

```diff
@@ -22,13 +37,56 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def removed_vowels(string)
+    string.gsub(/[aeiou]/, '') 
+end
+
```

Comment:
Nice use of RegEx here. What happens if you plug in `beatles_album3` into your method? Make sure you are taking capital letters into account!